### PR TITLE
Remove link to contacts for not logged in users

### LINKED ISSUE #381: Remove link to contacts for not logged-in users
This link below an article content should be displayed only for logged in users:

screen shot 2018-06-08 at 15 04 30

### PR DESCRIPTION
**Pull request by kantord**
fixes #381 <br/><br/><br/><url>LiveURL: https://smartfaq-remove-link-to-contacts-for-not-logged-in-users.surge.sh</url>

### CODE REVIEW

**Comment by kantord** [comment]
@git-toni what if I just put the entire thing inside ContactPageLink? It's only used in these 3 places:

```javascript
src/StaticFAQ/ArticleFeedback/FAQArticleFeedback.js:import { ContactPageLink } from '../../common';
src/StaticFAQ/StaticFAQError.js:import { ContactPageLink } from '../common';
src/SingleBookingPage/bookingItem/Contact.js:import ContactPageLink from '../../common/ContactPageLink';
```

**Comment by git-toni** [comment]
I'm definitely more inclined to place the render/dontRender logic in the parent rather than the child(ie. ContactPageLink), I think it's none of the child's concern. But maybe we need more opinions here :smile:

**Comment by kantord** [comment]
@git-toni hmm no, actually you're right :thinking: It's none of the child's concern indeed. But this gave me a new idea: if we shouldn't make it a prop, maybe we should make it a component. I'm thinking of creating a `<LoggedInOnly>` component. What do you think? That'd be basically a high-level wrapper that'd enable us to implement this kind of thing without importing `withUser` and without the ternary. That could be used to refactor stuff in several places, in order to hide the implementation detail for detecting logged-in status.

**Comment by git-toni** [comment]
@kantord I think that's a great idea :) Granted you use that component or HOC function at the parent level (ie. Replacing the ternary+withUser). 
Maybe a mix between `src/common/redirectsLoggedIn.js` and `src/common/integration/urlProcessing.js`? :)

File: src/StaticFAQ/StaticFAQError.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const StaticFAQError = () => (
       hesitate to contact us.
     </Text>
     <div className="contactLink">
-      <ContactPageLink />
+      <ContactPageLink available />
```
[the remark this answers is not in the document]
Comment (kantord) [reply src/StaticFAQ/StaticFAQError.js]: Should we hide the contact details from users when smart faq is down??

**Comment by kantord** [comment]
@git-toni what do you think of the implementation now? I think more components could be refactored this way, and if needed, a ` <UserStatus.LoggedOut>` could be created too, but I don't want to do that refactoring without being sure that this is a good implementation :smile:

**Comment by git-toni** [comment]
@kantord Looks great! Waiting for green pipeline to test + merge : D

**Comment by kantord** [comment]
Seems green now.

**Pull request merged**